### PR TITLE
feature: create RA function

### PR DESCRIPTION
**Pull request by CarlosDcomp**
### Feature Description
- Now when trying to Generate RA and not being successful, a dialog box
will appear and the user will be able to create an RA for the student.
- In the student form, a GOV ID field was added that will store the RA generated or created, being disabled
for editing and having a copy button if the user wants it.

Examples:
- GOV ID field
image

- Path to Create RA
image

### CODE REVIEW

File: app/modules/sedsp/usecases/GenRA.php [hunk 1]
```diff
@@ -14,34 +14,35 @@ class GenRA
...
+                $student = $ucadd->exec($id,$TryRA->getoutAluno()->outNumRA);
                 return $student->gov_id;
-            } else {
-                if($force) {
-                    return 'criar um novo..';
-                } else {
+            }else {
+                if($force){
+                    throw new Exception('RA Não Encontrado');
+                    $ucnewstudent = new AddStudentToSED();
```
Comment (igorgoncalves) [inline app/modules/sedsp/usecases/GenRA.php:32]: Esse código nunca é executado pq está após o throw. Remova o código

Comment (igorgoncalves) [inline app/modules/sedsp/usecases/GenRA.php:32]: Esse código nunca é executado pq está após o throw. Remova o código

**Review by igorgoncalves — CHANGES_REQUESTED** [review]

File: app/modules/sedsp/usecases/GenRA.php (hunk 1 above)
Comment (CarlosDcomp) [reply app/modules/sedsp/usecases/GenRA.php:32]: Remover esse else?

Comment (CarlosDcomp) [reply app/modules/sedsp/usecases/GenRA.php]: Feito, verifique se era isso mesmo

**Pull request merged**